scalafix-core/shared/src/main/scala/scalafix/v1/Doc.scala

```diff
@@ -0,0 +1,59 @@
+package scalafix.v1
```

Comment:
Am I right in thinking that scalafix.v0 is still the default? Or do you plan to flip the switch to use scalafix.v1 in 0.6-final?

Reply:
This PR makes `scalafix.v1` the default, the hard work was making it so seamless that 0 changes are required for existing rules that use `scalafix.v0`.

The cli and testkit support rules written against both v1 and v0.